packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -4,204 +4,549 @@
+          ),
+          bottomHeight: kBottomHeight,
+          top: Align(
+            alignment: Alignment.topRight,
+            child: Container(
+              padding: const EdgeInsets.all(4.0),
+              margin: const EdgeInsets.only(right: kRightWidth + kMargin),
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.start,
+                crossAxisAlignment: CrossAxisAlignment.end,
+                children: <Widget>[
+                  Text(
+                    'flex: ${node.flexFactor}',
+                    style: TextStyle(fontWeight: FontWeight.bold),
+                  ),
+                  if (flexFactor == 0 || flexFactor == null)
+                    Text(
+                      'unconstrained ${isRow ? 'horizontal' : 'vertical'}',
+                      style: TextStyle(
+                        color: ThemedColor(
+                          const Color(0xFFD08A29),
+                          Colors.orange.shade700,
+                        ),
+                        fontStyle: FontStyle.italic,
+                      ),
+                      maxLines: 2,
+                      softWrap: true,
+                      overflow: TextOverflow.ellipsis,
+                      textScaleFactor: 0.8,
+                      textAlign: TextAlign.right,
+                    ),
+                ],
               ),
-            ],
+            ),
           ),
-          direction: Axis.vertical,
         ),
-        height: length,
-        width: length * 0.25,
       ),
-      bottom: Container(
-        margin: const EdgeInsets.only(top: 16.0),
-        child: ArrowWrapper.bidirectional(
-          arrowColor: theme.splashColor,
-          arrowStrokeWidth: 1.0,
-          child: Column(
-            children: <Widget>[
-              Text(
-                // TODO(albertusangga): Use real width
-                '${length.round()} px',
-              ),
-              const Text('width'),
-            ],
+    );
+  }
+
+  Widget _buildAxisAlignmentDropdown(Axis axis) {
+    Color color;
+    String axisDescription;
+    List<Object> alignmentEnumEntries;
+    Object selected;
+    if (axis == Axis.horizontal) {
+      color = horizontalColor;
+      axisDescription = properties.horizontalDirectionDescription;
+    } else {
+      color = verticalColor;
+      axisDescription = properties.verticalDirectionDescription;
+    }
+    if (axis == direction) {
+      alignmentEnumEntries = MainAxisAlignment.values;
+      selected = mainAxisAlignment;
+    } else {
+      alignmentEnumEntries = CrossAxisAlignment.values;
+      selected = crossAxisAlignment;
+    }
+    return Container(
+      constraints: const BoxConstraints(maxWidth: kDropdownMaxWidth),
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.start,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        mainAxisSize: MainAxisSize.min,
+        children: <Widget>[
+          Text(
```

Comment:
maybe move the text so it is right aligned so there is less space between the text and image.

Reply:
The space between the text and image was caused by the Row mainAxisAlignment (spaceBetween).
I changed it to center for now and this is how it looks like:
![alignment_dropdown](https://user-images.githubusercontent.com/25892469/68825417-e6588c80-064e-11ea-9dc1-6a38c84e410b.gif)

Reply:
This is the right align version with 8.0 margin, what do you think?
![alignment_dropdown_2](https://user-images.githubusercontent.com/25892469/68825589-7696d180-064f-11ea-956d-3e6c161c67e7.gif)